modules/ui/map_in_map.js

```diff
@@ -1,327 +1,297 @@
+
+      updateprojection();
+      _gesture = null;
+      context.map().center(projection.invert(_cMini)); // recenter main map..
+    }
+
+
+    function updateprojection() {
+      var loc = context.map().center();
+      var tMain = context.projection.transform();
+      var zMain = geoScaleToZoom(tMain.k);
+      var zMini = Math.max(zMain - _zDiff, 0.5);
+      var kMini = geoZoomToScale(zMini);
+
+      projection.translate([tMain.x, tMain.y]).scale(kMini);
+
+      var point = projection.project(loc);
+      var mouse =
+        _gesture === 'pan'
+          ? vecSubtract([_tCurr.x, _tCurr.y], [_tStart.x, _tStart.y])
+          : [0, 0];
+      var xMini = _cMini[0] - point[0] + tMain.x + mouse[0];
+      var yMini = _cMini[1] - point[1] + tMain.y + mouse[1];
+
+      projection.translate([xMini, yMini]).dimensions([[0, 0], _dMini]);
+
+      _tCurr = projection.transform();
+
+      if (_isTransformed) {
+        utilSetTransform(tiles, 0, 0);
+        utilSetTransform(viewport, 0, 0);
+        _isTransformed = false;
+      }
+
+      zoom.scaleExtent([geoZoomToScale(0.5), geoZoomToScale(zMain - 3)]);
+
+      _skipEvents = true;
+      wrap.call(zoom.transform, _tCurr);
+      _skipEvents = false;
+    }
+
+
+    function redraw() {
+      clearTimeout(_timeoutID);
+      if (_isHidden) return;
+
+      updateprojection();
+      drawBoundingBox();
+    }
+
+
+    /**
+     * drawBoundingBox
+     * Simple bounding box draw assuming that there will only ever be the one bounding box
+     * inside the 'bbox' container. If there is no 'bbox' container on the minimap stage, it will create one.
+     */
+    function drawBoundingBox() {
+      var bbox = context.map().extent().bbox();
+
+      const topLeftPoint = projection.project([bbox.minX, bbox.maxY]);
+      const bottomRightPoint = projection.project([bbox.maxX, bbox.minY]);
+      const boxWidth = Math.abs(bottomRightPoint[0] - topLeftPoint[0]);
+      const boxHeight = Math.abs(bottomRightPoint[1] - topLeftPoint[1]);
+
+      let container = context.minipixi.stage;
+      container.name = 'minimap-stage';
+      const bboxContainer = container.getChildByName('bbox');
+      if (!bboxContainer) {
+        const bboxContainer = new PIXI.Container();
+        bboxContainer.name = 'bbox';
+        bboxContainer.interactiveChildren = false;
+        bboxContainer.buttonMode = false;
+        bboxContainer.interactive = false;
+        container.addChild(bboxContainer);
+
+        const bboxGraphic = new PIXI.Graphics()
+          .clear()
+          .lineStyle(2, 0x00ffff)
+          .drawRect(topLeftPoint[0], topLeftPoint[1], boxWidth, boxHeight);
+        bboxContainer.addChild(bboxGraphic);
+      } else {
+        const bboxPolyGraphic = bboxContainer.children[0];
+        bboxPolyGraphic
+          .clear()
+          .lineStyle(2, 0x00ffff)
+          .drawRect(topLeftPoint[0], topLeftPoint[1], boxWidth, boxHeight);
+      }
+    }
+
+
+    function queueRedraw() {
```

Comment:
Pretty sure I can get rid of some jank just by getting rid of the 'queueRedraw()'. Taking a look at this.

Reply:
Yep, toasted this method, works flawlessly